README.md

```diff
@@ -126,11 +126,22 @@ Role Variables
     osbs_source_registry_uri: https://source.registry.fedoraproject.org  # Empty default value
 
     # Pruning
-    osbs_prune: false
-    osbs_prune_schedule: '0 0 */8 * *'
-    osbs_prune_secret: ''
-    osbs_prune_image: ''
-    osbs_prune_commands: ["/prune.sh"]
+    osbs_pruner_image: openshift3/ose
+    osbs_pruner_command_build:
+    - /usr/bin/oc
+    - adm
+    - prune
+    - builds
+    - --orphans=true
+    - --confirm
+    osbs_pruner_schedule_build: "0 0 * * *"
+    # Automatically prunes pods from CronJobs
+    osbs_pruner_successful_jobs: 5
+    osbs_pruner_failed_jobs: 5
+    # Define this variable to enable pruning: will be created if absent
```

Comment:
What will be created?

Reply:
pruner service account ?

Reply:
I'll clarify as "# Define this variable to enable pruning: account will be created if absent"